File: Assignment-32/program4.java
import java.util.*;

class Program4
{
      public static void main(String arg[])
      {
            Scanner sobj = new Scanner(System.in);

            System.out.println("Enter The Size of Elements That You Want To Create:");
            int iSize = sobj.nextInt();
            
            ArrayDemo obj = new ArrayDemo(iSize);
            obj.Accept();

            obj.Display();
           
      }
}
class ArrayDemo
{
      public int Arr[];

      public ArrayDemo(int iSize)
      {
            Arr = new int[iSize];
      }
      public void Accept()
      {
            Scanner sobj = new Scanner(System.in);
            System.out.println("Enter The Elements:");
            for(int icnt = 0;icnt<Arr.length;icnt++)
            {
                  Arr[icnt] = sobj.nextInt();
            }
      }

      public void Display()
      {
            for(int icnt = 0;icnt<Arr.length;icnt++)
            {
                  if((Arr[icnt] % 3 == 0)&&(Arr[icnt] % 5 == 0))
                  {
                        System.out.print(Arr[icnt]+"\t");
                  }
                 
            }
           
      }
}

File: Assignment-26/program4.c
#include<stdio.h>

void strDigit(char *str)
{
     
      while(*str!='\0')
      {
            if(*str >= '0' && *str<= '9')
            {
                printf("%c\t",*str);
                
            }
            str++;
            
      }
}    
int main()
{
      char arr[20];
   
      printf("Enter The String :");
      scanf(" %[^'\n']s",arr);

      strDigit(arr);

      
      return 0;
}